R/MixedModelsCommon.R

```diff
@@ -178,19 +178,19 @@
   for(var in unlist(options$fixedEffects)) {
     if(is.factor(dataset[,.v(var)]) || is.character(dataset[,.v(var)])){
       if(length(unique(dataset[,.v(var)])) == nrow(dataset))
-        jaspBase:::.quitAnalysis(gettextf("The categorical fixed effect '%s' must have fewer levels than the overall number of observations.",var))
+        JASP:::.quitAnalysis(gettextf("The categorical fixed effect '%s' must have fewer levels than the overall number of observations.",var))
```

Comment:
You can remove all these `JASP:::`. It's now called jaspBase, but it's also exported and the modules have access to all exported functions without namespaces

Reply:
should I change it in stable as well then?

Reply:
No. These changes will only come into effect after 0.14 and we start using `development` instead of `stable`
Stable still uses `JASP` and `JASPgraphs` for this one release

Reply:
ok, understood